com.github.Cogitri.gxi.yml

```diff
@@ -0,0 +1,27 @@
+app-id: com.github.Cogitri.gxi
+runtime: org.freedesktop.Platform
+runtime-version: "18.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: gxi
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=home
```

Comment:
Actually, I don't think this is necessary, as I use GtkFileChooser for opening files. Will figure out why it doesn't show any files without this.